### PR TITLE
[Dominion] Fix profile tabs hrefs + map zoom

### PR DESCRIPTION
**Pull request by KhadijaMahanga**
## Description
This PR fixes profile tabs hrefs for article section in profile_detail page, and only displays profile tabs for existing article section for a profile. It also zooms in featured geography below country level.

Fixes # (issue)
- [x] Bug fix (non-breaking change which fixes an issue)

## Screenshots
Screenshot from 2019-04-01 09-06-35
Screenshot from 2019-04-01 09-05-39
Screenshot from 2019-04-01 09-02-25

## Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation

### CODE REVIEW

**Comment by karimkawambwa** [comment]
> Please include a summary of the change and which issue is fixed. Please also include relevant motivation and context. List any dependencies that are required for this change.

@KhadijaMahanga

File: dominion/static/js/profile_map_mapit.js [hunk 1]
```diff
@@ -129,7 +129,8 @@ var ProfileMaps = function() {
             style: self.featureGeoStyle,
         });
         this.map.addLayer(layer);
-        this.map.setView(this.dominion_country.centre, this.dominion_country.zoom);
+        this.map.fitBounds(layer.getBounds());
+      //  this.map.setView(this.dominion_country.centre, this.dominion_country.zoom);
```
Comment (karimkawambwa) [inline dominion/static/js/profile_map_mapit.js]: remove this commented out code if its not useful anymore

**Review by karimkawambwa — COMMENTED** [review]
What sort of scroll behaviour are you going for ?
I don't see smooth scrolling and the tabs are hidden at the top when scrolling to profile section so you have to scroll all the way back.

**Review by DavidLemayian — CHANGES_REQUESTED** [review]
@KhadijaMahanga Please add description and screenshots.

**Review by karimkawambwa — APPROVED** [review]
From the description, works as intended 🚢 it.

**Review by kilemensi — APPROVED** [review]

**Pull request merged**